certified-connectors/SmartCOMM On Demand Doc-Gen/script.csx

```diff
@@ -0,0 +1,17 @@
+﻿public class Script: ScriptBase {
+
+  public override async Task < HttpResponseMessage > ExecuteAsync() {
+	await this.UpdateRequest().ConfigureAwait(false);
+	var response = await this.Context.SendAsync(this.Context.Request, this.CancellationToken).ConfigureAwait(false);
+	return response;
+  }
+
+  private async Task UpdateRequest() {
+    var contentAsString = await this.Context.Request.Content.ReadAsStringAsync().ConfigureAwait(false);
+    var contentAsJson = JObject.Parse(contentAsString);
+    var transactionData = (string) contentAsJson["transactionData"];
+	var transactionDataBytes = System.Text.Encoding.UTF8.GetBytes(transactionData);
```

Comment:
@tajujsag can you please fix the indentation for this one?

Reply:
I reformatted the whole code